src/DynamoCore/Core/IDSDKManager.cs

```diff
@@ -253,6 +255,11 @@ private bool Deinitialize()
             }
             return false;
         }
+        public void Dispose()
```

Comment:
making something `IDisposable` won't magically make it get disposed - you have to call this method from somewhere, like DynamoModel shutdown.

Reply:
added it